global.json

```diff
@@ -1,7 +1,7 @@
 {
   "sdk": {
-    "version": "7.0.100",
-    "allowPrerelease": false,
+    "version": "8.0.100-rc.1",
```

Comment:
Find out if the rc part can be removed, since it will mean another change once .NET 8 is released

Reply:
Can't fix I think, will see with rc.2 what happens, I imagine that it will roll into rc.2 automatically.